schema/schema_template.py

```diff
@@ -182,24 +183,16 @@ def validate_description(self, key, value):
 
 # -------------------------------------------------------------------------------------------------------------------
 # Hard-coded enumerations
-class Regime(enum.Enum):
+class Regime(Base):
```

Comment:
If you change Regime -> Regimes in the table name, it may make sense to also change it in the Class name for consistency.